src/Explorer/Controls/NotebookGallery/CodeOfConductComponent.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { shallow } from "enzyme";
+import * as sinon from "sinon";
+import React from "react";
+import { CodeOfConductComponent, CodeOfConductComponentProps } from "./CodeOfConductComponent";
+import { IJunoResponse, JunoClient } from "../../../Juno/JunoClient";
+import { HttpStatusCodes } from "../../../Common/Constants";
+
+sinon.stub(JunoClient.prototype, "acceptCodeOfConduct").returns({
```

Comment:
Might want to set this up in a separate sinon sandbox so this does not affect other tests